dev/com.ibm.ws.cdi.ejb_fat/fat/src/com/ibm/ws/cdi/ejb/apps/newTimer/TestEjbTimerServlet.java

```diff
@@ -0,0 +1,72 @@
+ /*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.cdi.ejb.apps.newtimer;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.IOException;
+
+import javax.ejb.EJB;
+import javax.servlet.ServletException;
+import javax.servlet.ServletOutputStream;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import componenttest.app.FATServlet;
+import org.junit.Test;
+
+@WebServlet("/Timer")
+public class TestEjbTimerServlet extends FATServlet {
+    private static final long serialVersionUID = 1L;
+    @EJB
+    EjbSessionBeanLocal bean;
+
+    private boolean needToSetUp = true;
```

Comment:
I'm not sure that the servlet lifecycle guarantees that every invocation of this servlet will use the same instance. Maybe make this field static to be sure we won't repeat the setup?